packages/botonic-plugin-contentful/README.md

```diff
@@ -181,11 +191,11 @@ import * as cms from '@botonic/plugin-contentful';
 ....
 
  {
-    payload: cms.ContentCallback.regexForModel(cms.ModelType.TEXT),
+    payload: cms.ContentCallback.regexForModel(cms.TopContentType.TEXT),
```

Comment:
thank you for spotting this. I'd rather use cms.ContentType instead since it's shorter and simpler, but not a biggie